libs/langchain/langchain/chains/structured_output/base.py

```diff
@@ -299,33 +306,46 @@ class Dog(BaseModel):
                 chain = create_structured_output_runnable(Dog, llm, prompt, mode="openai-json")
                 chain.invoke({"input": "Harry was a chubby brown beagle who loved chicken"})
     """  # noqa: E501
+    # for backwards compatibility
```

Comment:
i don't think we want to support this outside of openai-functions?

Reply:
I can push inside, but I usually handle this at top scope, even better would be to fix this via a function level decorator to add a parameter renamer